modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/_diffs/plugins/insertbutton/plugin.js

```diff
@@ -121,18 +127,24 @@
 			return instance._tableToolbar;
 		},
 
-		_createToolbar() {
-			if (this._toolbar) {
-				return;
-			}
+		_createToolbar(editor) {
 
-			const instance = this;
+			// Can be it or just disable this conditional. It is not preventing to
+			// create a new toolbar everytime We open a new one but it's necessary
+			// for avoiding referencing an old toolbar with old params
 
-			const editor = instance.editor;
+			// TODO - Could We try using `this._toolbar.refresh`?
+
+			if (
+				this._toolbar &&
+				this._toolbar._view.editor.name === editor.name
+			) {
+				return;
```

Comment:
This code is intended to prevent duplications of this._toolbar instances but was breaking use cases with multiple editors in the same page(with different editor.name)